### ISSUE TITLE
More developers wanted

### ISSUE THREAD

**Issue by jennylia** [jennylia/partner#1]
Hi All,

Would more developers like to work on this study partner platform built in meteor for hacktoberfest?

We still need lots of work to be done.
Some areas:
Css
Make a good durable backend
Documentations
Project Planning
Thanks,
Jenny

**Comment by jennylia** [comment]
@Hacktoberfest

**Comment by h3ct0rjs** [comment]
@jennylia  Do you need any translation to Spanish?  maybe I can help you. I was testing the link and it seems down from my site can you check this please.

**Comment by anumpatel** [comment]
@jennylia I am interested in CSS and backend part! and yeah as per @heticor915  site is down.

**Comment by hiteshgarg14** [comment]
Let me know if you are interested in doing this project with [Django](https://www.djangoproject.com/).

**Comment by jennylia** [comment]
@heticor915 Yes please!! If you like please help me write a short description in Spanish in README.md and submit a PR 👍 . I'll merge it in once you did that. If you like the concept, please give it a star.

**Comment by jennylia** [comment]
@anumpatel @heticor915 Yes CSS would be great! If you want to start by making some CSS changes, even just make a consistent color scheme, and submit a PR that would be great!  I'll merge it in once you did that. If you like the concept of finding study partners, please give it a star. You guys are right, the link is down. I was using bluemix during a hackathon event with free credit, guess that ran out. If you can think of free deployment solutions with Meteor, please do so and do a PR and I'll merge it.

**Comment by jennylia** [comment]
@hiteshgarg14 YES PLEASE!
Please take this concept and implement it in Django. If you like the concept, please give this repo a star.
Please include some description to Django project in README and submit a PR and I'll merge it in.

**Comment by gdsrosa** [comment]
When I start? I can help with CSS!

**Comment by h3ct0rjs** [comment]
@hiteshgarg14  After making a few test I think we could try write something useful using Django @hiteshgarg14  Are you able to write this, I will review your code if you want before we do the merge.
@jennylia Sure I'd like to add the spanish translation :smile:

**Comment by h3ct0rjs** [comment]
@jennylia Where exactly are the locations files? What should I translate ?.  You will need to make a new deployment using the bluemix or another cloud

**Comment by Mintux92** [comment]
@hacktoberfest

**Comment by Marlysson** [comment]
This project still is active?

Which features are planning?
